scripts/update-drafts.ts

```diff
@@ -178,7 +196,43 @@ async function main() {
 
       feature.comment = usedFeaturesComment.trimEnd();
     }
-    await fs.writeFile(`features/draft/spec/${id}.yml`, feature.toString());
+
+    const destination = `features/draft/spec/${id}.yml`;
+    const proposedFile = feature.toString();
+
+    let originalFile: string;
+    try {
+      originalFile = await fs.readFile(destination, { encoding: "utf-8" });
+    } catch (err: unknown) {
+      // If there's no file for this spec already, write a new one immediately.
+      if (typeof err === "object" && "code" in err && err.code === "ENOENT") {
+        await fs.writeFile(destination, proposedFile);
+        logger.info(`${destination}: new spec file added`);
+        continue;
+      }
+      throw err;
+    }
+
+    // If there's a file for this spec already, write updates only if something
+    // other than the `draft_date` changed. Because changes can be comments, we
+    // have to check a diff rather than parsing and comparing values.
+    const changes = diff.diffLines(originalFile, proposedFile);
+    const noChanges =
+      changes.length === 1 && !changes[0].added && !changes[0].removed;
+    const onlyDateChanges =
+      !noChanges &&
+      changes.length === 3 &&
```

Comment:
document the magic 1 and 3 numbers above

Reply:
Good idea! I won't even remember that in a week. 😅 Fixed with 13e94dbc.